### PR TITLE
Add parameter which only for tag to status command

### PR DESCRIPTION
**Pull request by kitakitabauer**
### Checklist
- [x] All new jobs, commands, executors, parameters have descriptions
- [x] Examples have been added for any significant new features
- [x] README has been updated, if necessary

### Description
Added a new parameter - only_for_tag - to the slack/status command, which allows slack updates to be triggered for builds on any tag only.

### CODE REVIEW

File: src/commands/status.yml [hunk 1]
```diff
@@ -49,7 +54,9 @@ steps:
       shell: /bin/bash
       when: always
       command: |
-        if [ "x" == "x<< parameters.only_for_branch>>" ] || [ "${CIRCLE_BRANCH}" == "<< parameters.only_for_branch>>" ]; then
+        if [[ "x" == "x<< parameters.only_for_branch >>" && "${CIRCLE_BRANCH}" == "<< parameters.only_for_branch >>" ]] ||
+           << parameters.only_for_tag >> && [ -n "${CIRCLE_TAG}" ] ||
```
Comment (eddiewebb) [inline src/commands/status.yml]: I didn't think bash will evaluate this as true/false, and needs to check for matching the literal string "true" or false is empty..

**Comment by iynere** [comment]
@kitakitabauer thanks, this is a good feature add. i have some thoughts about doing this in a more agnostic way, so let me take a stab at this. i may push my own branch & PR—if so, i'll CC you on it so you can 👀

**Comment by kitakitabauer** [comment]
@iynere thanks, i'm waiting for your PR!

File: src/commands/status.yml (hunk 1 above)
Comment (kitakitabauer) [reply src/commands/status.yml]: thanks! condition changed. d00f869

**Comment by kitakitabauer** [comment]
@iynere When will you be able to PR? This function is necessary as soon as possible.

**Comment by iynere** [comment]
@kitakitabauer sorry, was working on a higher-level solution for this that would be applicable to any orb, but it got complicated ! i'll prioritize merging a specific solution for this issue over my broader implementation, thanks for the ping !

**Comment by iynere** [comment]
@kitakitabauer i ended up implementing my solution—can you refactor this PR to use it?

i've added a command to circleci's build-tools-orb that allows you to run any arbitrary command or set of commands only on certain branches or tags:

https://circleci.com/orbs/registry/orb/circleci/build-tools#commands-run-filtered

with that, you should be able to do what you're trying to do here

**Comment by KyleTryon** [comment]
+1 love the solution implemented here. Should be highly extendable.

**Comment by KyleTryon** [comment]
Outdated.

**Pull request closed without merging**